content/includes/footer.html

```diff
@@ -121,6 +121,15 @@ <h4 class="va-crisis-panel-body-title">Get help from Veterans Crisis Line</h4>
 “To care for him who shall have borne the battle and for his widow, and his orphan.”
 - Abraham Lincoln
 -->
-
+{% if widgets %}
+<script type="text/javascript">
+(function() {
+  var widgets = {{ widgets | json }};
```

Comment:
~Is this supposed to be `|` or `||`? As `|`, it confuses me.~ Forgot this file was subject to liquid templating. That's not confusing at all! 🙄